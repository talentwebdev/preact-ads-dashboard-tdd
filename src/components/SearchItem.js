import React from "react";
import SearchIcon from "@material-ui/icons/Search";
function SearchItem({ title, name }) {
  return (
    <div className="searchItem">
      <div id="itemTitle">
        <p>{title}</p>
      </div>
      <div id="itemBody">
        <SearchIcon />
        <h3>{name}</h3>
      </div>
    </div>
  );
}

export default SearchItem;
