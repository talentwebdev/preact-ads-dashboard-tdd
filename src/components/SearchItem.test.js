import { shallow } from "enzyme";
import SearchItem from "./SearchItem";

describe("SearchItem", () => {
  let searchItemContainer;
  beforeEach(() => (searchItemContainer = shallow(<SearchItem />)));
  it("should render the div element", () => {
    expect(searchItemContainer.find("div").length).toBeGreaterThanOrEqual(1);
  });
  it("renders SearchItem correctly", () => {
    expect(searchItemContainer).toMatchSnapshot();
  });
});
