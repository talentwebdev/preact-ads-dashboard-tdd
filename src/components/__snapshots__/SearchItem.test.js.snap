// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchItem renders SearchItem correctly 1`] = `ShallowWrapper {}`;
